pkg/ytconfig/names.go

```diff
@@ -188,3 +188,20 @@ func (g *Generator) FormatComponentStringWithDefault(base string, name string) s
 	}
 	return base
 }
+
+func (g *Generator) GetMasterCachesStatefulSetName() string {
+	return g.getName("ms-cache")
```

Comment:
I suggest "msc" since others are rather short
```
yt                 hp                                                 147/155   95d
yt                 ms                                                 3/3       95d
yt                 qa                                                 3/3       95d
yt                 qt                                                 3/3       80d
yt                 rp                                                 147/155   95d
yt                 sch                                                3/3       95d
yt                 tnd                                                147/155   95d
yt                 yqla                                               3/3       80d
```

Reply:
Yep, agree